configs/inspection/Square.xml

```diff
@@ -2,5 +2,6 @@
 <inspections version="1.0" is_locked="false">
   <option name="myName" value="Square" />
   <inspection_tool class="UnnecessarySemicolon" enabled="true" level="ERROR" enabled_by_default="true" />
+  <inspection_tool class="FallthruInSwitchStatement" enabled="true" level="ERROR" enabled_by_default="true" />
```

Comment:
Yeah, good point, I'll do that. Do you know if GitHub will correctly handle me amending and force-pushing this commit?